### ISSUE TITLE
Allow to freeze certain targets

### ISSUE THREAD

**Issue by windisch** [boschglobal/luisy#24]
What do you think about implementation of a `@luisy.freeze_output` decorator, which blocks the output once created from being automatically removed?

@ts933, @niclash91, @GANdalf2357: What do you think?

**Comment by ts933** [comment]
@windisch in which scenarios will that be helpful? so that someone doesn't delete something by accident when re-runing a task?

**Comment by windisch** [comment]
> @windisch in which scenarios will that be helpful? so that someone doesn't delete something by accident when re-runing a task?

Exactly! That way, an important (or computationally very expensive) output can be saved to not be overwritten accidentially.
